_posts/2019-07-29-plonk-stack.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Introducing the PLONK Stack for Cloud Native Developers"
+description: You've heard of LAMP, JAM, and MEAN, but what's the PLONK stack, and why should you be considering it for your Cloud Native Applications?
+date: 2019-08-04
+image: /images/2019-intro-plonk/map-reader.jpg
+categories:
+  - community
+  - end-user
+  - production
+  - kubernetes
+  - plonk
+  - cloud native
+author_staff_member: alex
+dark_background: true
+---
+
+You've heard of LAMP, JAM, and MEAN, but what's the PLONK stack, and why should you be considering it for your Cloud Native Applications?
```

Comment:
as above

Reply:
> One general point I'd suggest you consider is counting how many times 'I' has been used versus 'we' and consider whether that is consistent with the messaging around community and contributors.

Whilst I did used to use the word "we" everywhere, I've stopped. Here is why: one piece of key feedback I had from VMware's media team was not to write in the voice of "we", especially for a blog post written in my own voice.

Reply:
What I'll do is add the following into the intro to make it clear that I'm writing as my voice:

>  In this blog post I'll walk you through each project and add details from my experience developing OpenFaaS with the community since 2016.